From ef764d96b30ef1261416cd17d310587fd357e2ca Mon Sep 17 00:00:00 2001
From: Josiah Schwab <jschwab@gmail.com>
Date: Tue, 5 Jan 2016 14:07:17 -0800
Subject: [PATCH 2/4] Correct weak transitions in test_suite

The input nuclear data in the test_suite examples of the special weak
rates capabilities mis-used ft-values.  The identical ft-value was used
for both the electron-capture and beta-decay transition between two
states.  However, these ft-values should differ by the ratio of the spin
degeneracy of the states.

This error origininated in Schwab et al. (2015) and propigated through
into Paxton et al. (2015).  Within their range of validity, these rates
are now in agreement with those of Suzkui et al. (2016).
---
 star/test_suite/8.8M_urca/urca.transitions | 43 +++++++++++++++++-------------
 star/test_suite/wd_aic/aic.transitions     | 42 ++++++++++++++++-------------
 2 files changed, 49 insertions(+), 36 deletions(-)

diff --git a/star/test_suite/8.8M_urca/urca.transitions b/star/test_suite/8.8M_urca/urca.transitions
index d34d608..fc5ccd9 100644
--- a/star/test_suite/8.8M_urca/urca.transitions
+++ b/star/test_suite/8.8M_urca/urca.transitions
@@ -1,46 +1,53 @@
 ! these are the transitiions for electron capture / beta decay
 ! these values are drawn from the A = 23, 25, 27 nuclear data sheets
+
 ! the format is
 ! <iso1> <iso2> <ntrans>
 ! <si> <sf> <logft>
 
 ! A = 23: ne <-> na
 
+! (0.000, 5/2+) <-> (0.000, 3/2+)
+! (0.000, 5/2+) <-> (0.440, 5/2+)
+
 ne23 na23 2
 1 1 5.27
 1 2 5.38
-!1 3 5.82
-!1 4 6.13
 
 na23 ne23 2
-1 1 5.27
+1 1 5.09
 2 1 5.38
-!3 1 5.82
-!4 1 6.13
 
 
 ! A = 25: na <-> mg
 
-na25 mg25 3
-1 1 5.26
-1 3 5.04
-1 4 5.03
-!1 5 6.04
+! (0.000, 5/2+) <-> (0.000, 5/2+)
+! (0.090, 3/2+) <-> (0.000, 5/2+) -- not in NDS (set to 5.0)
+! (0.000, 5/2+) <-> (0.975, 3/2+)
+! (0.000, 5/2+) <-> (1.612, 7/2+)
 
-mg25 na25 3
+na25 mg25 2
 1 1 5.26
-3 1 5.04
-4 1 5.03
-!5 1 6.04
+2 1 5.0
+!1 3 5.04
+!1 4 5.03
 
+mg25 na25 2
+1 1 5.26
+1 2 5.2
+!3 1 4.86
+!4 1 5.15
 
 ! A = 27: mg <-> al
 
+! (0.000, 1/2+) <-> (0.844, 1/2+)
+! (0.000, 1/2+) <-> (1.015, 3/2+)
+
 mg27 al27 2
-1 2 4.7297
-1 3 4.9340
+1 2 4.73
+2 1 5.50
 
 al27 mg27 2
-2 1 4.7297
-3 1 4.9340
+2 1 4.73
+1 2 5.68
 
diff --git a/star/test_suite/wd_aic/aic.transitions b/star/test_suite/wd_aic/aic.transitions
index 5db7604..0e11690 100644
--- a/star/test_suite/wd_aic/aic.transitions
+++ b/star/test_suite/wd_aic/aic.transitions
@@ -1,5 +1,6 @@
 ! these are the transitiions for electron capture / beta decay
-! these values are from Martinez-Pinedo (2014)
+! the ft-values are derived from Martinez-Pinedo et al. (2014)
+
 ! the format is
 ! <iso1> <iso2> <ntrans>
 ! <si> <sf> <logft>
@@ -9,59 +10,64 @@
 
 ! (1.634 2+) -> (0.000 2+)
 ! (0.000 0+) -> (1.057 1+)
+! (0.000 0+) -> (0.000 2+) - exp. upper limit
 ne20 f20  2
-2 1 4.970
-1 5 4.380
+2 1 4.97
+1 5 4.38
+!1 1 9.80
 
 ! (0.000 2+) -> (1.674 2+)
 ! (1.057 1+) -> (0.000 0+)
 f20  o20  2
-1 2 5.429
-5 1 4.211
+1 2 5.43
+5 1 4.21
 
 ! A = 24: mg -> na -> ne
 
 ! (0.000 0+) -> (0.472 1+)
 ! (0.000 0+) -> (1.347 1+)
 mg24 na24 2
-1 2 4.815
+1 2 4.8
 1 6 3.772
 
 ! (0.000 4+) -> (3.972 4+)
 ! (0.000 4+) -> (4.866 3+)
 ! (0.472 1+) -> (0.000 0+)
 na24 ne24 3
-1 4 6.209
-1 6 4.423
-2 1 4.829
+1 4 6.2
+1 6 4.42
+2 1 4.8
 
 ! beta decay transitions
 ! A = 20: o -> f -> ne
 
 ! (0.000 2+) -> (1.634 2+)
 ! (1.057 1+) -> (0.000 0+)
+! (0.000 2+) -> (0.000 0+) - exp. upper limit
 f20  ne20 2
-1 2 4.970
-5 1 4.380
+1 2 4.97
+5 1 4.86
+!1 1 10.5
 
 ! (1.674 2+) -> (0.000 2+)
 ! (0.000 0+) -> (1.057 1+)
 o20  f20  2
-2 1 5.429
-1 5 4.211
+2 1 5.43
+1 5 3.73
 
 ! A = 24: ne -> na -> mg
 ! (0.472 1+) -> (0.000 0+)
 ! (1.347 1+) -> (0.000 0+)
 
 na24 mg24 2
-2 1 4.815
-6 1 3.772
+2 1 5.3
+6 1 4.249
 
 ! (3.972 4+) -> (0.000 4+)
 ! (4.866 3+) -> (0.000 4+)
 ! (0.000 0+) -> (0.472 1+)
+
 ne24 na24 3
-4 1 6.209
-6 1 4.423
-1 2 4.829
+1 2 4.4
+4 1 6.2
+6 1 4.31
-- 
2.9.2

